packages/shady-css-scoped-element/tests/shady-css-scoped.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<!--
+    @license
+    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+<head>
+  <title>HTML Imports Dynamic Elements</title>
+  <script src="../node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
+  <script src="../shady-css-scoped-element.min.js"></script>
+  <script>WCT = {waitFor: function(cb){cb()}};</script>
+  <script src="../node_modules/wct-browser-legacy/browser.js"></script>
+</head>
+<body>
+  <shady-css-scoped>
+    <style>
+      html{
+        --foo: #333333;
+      }
+    </style>
+  </shady-css-scoped>
+  <template>
+    <style>
+      div {
+        background-color: var(--foo);
+      }
+    </style>
+    <div id="test-div"></div>
+  </template>
+  <my-element></my-element>
+
+  <script>
+    suite('shady-css-scoped-element', function() {
+      setup(function() {
```

Comment:
Use `suiteSetup`. `setup` runs before every test, so future tests will fail trying to define this element multiple times.